util/topgen/merge.py

```diff
@@ -580,6 +580,14 @@ def _get_clock_group_name(clk: Union[str, OrderedDict],
     return group_name, src_name
 
 
+def is_unmanaged_clock(top: OrderedDict, clock: str):
+    clk = top['unmanaged_clocks']._asdict().get(clock)
```

Comment:
The body of this function could be one line as `return clock in top['unmanaged_clocks']._asdict()`